shell/test/specs/demo-tests.js

```diff
@@ -335,7 +343,7 @@ function _waitForAndMaybeAcceptSuggestion(substring, accept) {
       throw e;
     }
   };
-  browser.waitUntil(findSuggestion, 5000, `couldn't find suggestion ${substring}`);
+  browser.waitUntil(findSuggestion, 50000, `timed out looking find suggestion '${substring}'.`);
```

Comment:
Some concern that this could cause our Travis jobs to hang or otherwise run for much longer than they do today?

Reply:
I put back the old value (and fixed the grammar in the message).